attribute.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+# Represents a classroom.
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    students << student
+    student.classroom = self
+  end
+end
+
+# Represents a Student.
+class Student
+  attr_accessor :name, :classroom
+
+  def initialize(name)
+    @name = name
+    @classroom = nil
+  end
+end
```

Comment:
- [ ] Instead of redefining a second student class here I'd suggest you use the class in the `student.rb` file and add a method that serves as the 'belongs-to' side (a student belongs to a classroom) that will add a classroom for a student and make sure to add the proper attribute reader for the classroom instance variable.